.eslintrc.json

```diff
@@ -0,0 +1,19 @@
+{
```

Comment:
All comments in here should be removed. Especially when committing little and often git is a great time machine that will show you what was added and when so you never need to add notes like this as they will inevitably create a lot of unnecessary clutter